spec/lib/myfinance/resources/entity_spec.rb

```diff
@@ -60,4 +60,107 @@
       end
     end
   end
+
+  describe "#get_new", vcr: true do
+    context "when success" do
+      subject { client.entities.get_new }
+
+      it "returns empty entity" do
+        expect(subject.class).to eq(Myfinance::Entities::Entity)
+        expect(subject.id).to be_nil
+        expect(subject.name).to be_nil
+        expect(subject.account_id).to eq(3613)
+        expect(subject.charging_uuid).to be_nil
+        expect(subject.created_at).to be_nil
+        expect(subject.default_in_menu).to be_falsy
+        expect(subject.deleted_at).to be_nil
+        expect(subject.federation_subscription_number).to be_nil
+        expect(subject.imported_from_sync_at).to be_nil
+        expect(subject.updated_at).to be_nil
+      end
+    end
+
+    context "when not found" do
+      let(:client) { Myfinance.client('') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.